tests/parser/types/test_dynamic_array.py

```diff
@@ -1222,6 +1222,174 @@ def test_append_pop_complex(get_contract, assert_tx_failed, code_template, check
         """
         code = struct_def + "\n" + code
 
+    c = get_contract(code)
+    expected_result = check_result(test_data)
+    if expected_result is None:
+        # None is sentinel to indicatecool, I'd txn should revert
```

Comment:
what does this comment mean?

Reply:
oops, I have been getting rekt by middle click paste on my new mouse. "cool, I'd " should not be there.